<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpaceSwitcherConfiguration</class>
 <widget class="QWidget" name="SpaceSwitcherConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="vl_top">
       <item>
        <layout class="QHBoxLayout" name="hl_selectedControl">
         <item>
          <widget class="QLabel" name="lbl_selectedControl">
           <property name="text">
            <string>Selected Control: None</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_refresh">
           <property name="text">
            <string>Refresh</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hl_jsonContents">
         <item>
          <widget class="QLabel" name="lbl_jsonContents">
           <property name="text">
            <string>ILLMayaSpaceSwitcherConfig attribute JSON</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_generateDefaultJsonContents">
           <property name="text">
            <string>Generate Sample JSON</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_generateAutoJsonContents">
           <property name="toolTip">
            <string>This makes an attempt to auto generate some JSON that can be used as a base. It makes a best guess based on the name of the control and the names of the attributes already configured.</string>
           </property>
           <property name="text">
            <string>Auto Generate JSON</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="te_jsonContents">
         <property name="font">
          <font>
           <family>Lucida Console</family>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::LineWrapMode::NoWrap</enum>
         </property>
         <property name="plainText">
          <string/>
         </property>
         <property name="tabStopDistance">
          <double>40.000000000000000</double>
         </property>
         <property name="placeholderText">
          <string>JSON goes here</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hl_set">
         <item>
          <widget class="QPushButton" name="btn_validate">
           <property name="text">
            <string>Validate JSON</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_set">
           <property name="text">
            <string>Set Space Configuration on Control</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hl_selectedObject">
         <item>
          <widget class="QLabel" name="lbl_selectedObjectName">
           <property name="text">
            <string>Selected Object Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_getSelectedObjectName">
           <property name="toolTip">
            <string>Push this to set the below text box to the name of the selected object. You can copy paste that into the configutarion JSON above.</string>
           </property>
           <property name="text">
            <string>Get Selected Object Name</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="le_selectionName">
         <property name="font">
          <font>
           <family>Lucida Console</family>
          </font>
         </property>
         <property name="placeholderText">
          <string>Selected Object Name (Copy paste from here to the JSON)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <layout class="QVBoxLayout" name="vl_bottom">
       <item>
        <layout class="QHBoxLayout" name="hl_selectedControlAttributes">
         <item>
          <widget class="QLabel" name="lbl_selectedControlAttributes">
           <property name="text">
            <string>Selected Control Attributes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_getSelectedControlAttributes">
           <property name="text">
            <string>Get Selected Control Attributes</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="te_selectedControlAttributes">
         <property name="font">
          <font>
           <family>Lucida Console</family>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::LineWrapMode::NoWrap</enum>
         </property>
         <property name="plainText">
          <string/>
         </property>
         <property name="tabStopDistance">
          <double>40.000000000000000</double>
         </property>
         <property name="placeholderText">
          <string>Selected Control Attributes (Copy paste from here to the JSON)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
